boa/wrappers/__init__.py

```diff
@@ -14,7 +14,7 @@
 * :meth:`.BaseWrapper.run_model`: Defines how to run the model.
 * :meth:`.BaseWrapper.set_trial_status`: Defines how to determine the status of a trial (i.e., if the model
   run is completed, still running, failed, etc).
-* :meth:`.BaseWrapper.fetch_trial_data`: Retrieves the trial data (i.e., model outputs) and prepares it for
+* :meth:`.BaseWrapper.fetch_trial_data_single`: Retrieves the trial data (i.e., model outputs) and prepares it for
```

Comment:
Is it possible to merge all of the fetch trial data functions to a single function?